main.go

```diff
@@ -366,11 +295,11 @@ This means that when the API changes
 	}
 
 	envs := []string{
-		fmt.Sprintf("DELIVER_PASSWORD=%s", configs.Password),
+		fmt.Sprintf("DELIVER_PASSWORD=%s", string(configs.Password)),
```

Comment:
this is shorter:
```go
"DELIVER_PASSWORD="+string(configs.Password),
```

Also when you use a string formatter you use more than 100+ lines of code to "add" strings together.

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/228861e85a97e1f9399ec819aed8dc1e70e2ef4e